libs/build_wgrib2.sh

```diff
@@ -0,0 +1,117 @@
+#!/bin/bash
+
+set -eux
+
+name="wgrib2"
+version=${1:-${STACK_wgrib2_version}}
+
+software=$name-$version
+
+# Hyphenated version used for install prefix
+compiler=$(echo $HPC_COMPILER | sed 's/\//-/g')
+
+if $MODULES; then
+    set +x
+    source $MODULESHOME/init/bash
+    module load hpc-$HPC_COMPILER
+    module list
+    set -x
+    prefix="${PREFIX:-"/opt/modules"}/$compiler/$name/$version"
+    if [[ -d $prefix ]]; then
+        [[ $OVERWRITE =~ [yYtT] ]] && ( echo "WARNING: $prefix EXISTS: OVERWRITING!";$SUDO rm -rf $prefix ) \
+            || ( echo "WARNING: $prefix EXISTS, SKIPPING"; exit 1 )
+    fi
+else
+    prefix=${WGRIB2_ROOT:-"/usr/local"}
+fi
+
+export FC=$SERIAL_FC
+export CC=$SERIAL_CC
+export CXX=$SERIAL_CXX
+
+cd ${HPC_STACK_ROOT}/${PKGDIR:-"pkg"}
+
+URL="https://www.ftp.cpc.ncep.noaa.gov/wd51we/wgrib2/wgrib2.tgz.v${version}"
+
+[[ -d $software ]] || ( $WGET $URL; tar -xf wgrib2.tgz.v${version} )
+# wgrib2 is untarred as 'grib2'. Give a name with version.
+[[ -d $software ]] || mkdir $software && tar -xf wgrib2.tgz.v${version} -C $software --strip-components 1
+[[ ${DOWNLOAD_ONLY} =~ [yYtT] ]] && exit 0
+
+[[ -d $software ]] && cd $software || ( echo "$software does not exist, ABORT!"; exit 1 )
+
+# The Jasper inside of wgrib2 does not build with Clang on macOS
+# Implicit function declaration error and no way to pass flags to suppress it.
+host=$(uname -s)
+if [[ "$host" == "Darwin" ]]; then
+    if [[ `$CC --version` == *"clang"* ]]; then
+        echo "Warning: The Jasper contained in wgrib2 does not build with Clang on macOS"
+        export STACK_wgrib2_jasper=0
+    fi
+fi
+
+COMPILER=$(echo $HPC_COMPILER | cut -d/ -f1)
+if [[ "$COMPILER" == "intel" || "$COMPILER" == "ips" ]]; then
+    export COMP_SYS=intel_linux
+fi
```

Comment:
Instead of doing this, can we do:
```suggestion
if [[ $($CC --version | grep Intel) ]]; then
    export COMP_SYS=intel_linux
fi
```